easybuild/easyconfigs/c/CMake/CMake-3.6.1-foss-2015b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.6.1'
+
+homepage = 'http://www.cmake.org'
+description = """CMake, the cross-platform, open-source build system.
+ CMake is a family of tools designed to build, test and package software."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
Checksum Missing
```
umcg-mterpstra@calculon:~$ md5sum  /apps/sources/c/CMake/cmake-3.6.1.tar.gz 
d6dd661380adacdb12f41b926ec99545  /apps/sources/c/CMake/cmake-3.6.1.tar.gz
```

Ps other minor comments can be ignored but because i found this major comment i also included minor comments